src/shared/FunctionalValidation.Tests/FakeTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using CSharpFunctionalExtensions;
+using Machine.Fakes;
+using Machine.Specifications;
+using Result = CSharpFunctionalExtensions.Result;
+
+namespace FunctionalValidation.Tests;
+
+// ReSharper disable InconsistentNaming
+// ReSharper disable ArrangeTypeMemberModifiers
```

Comment:
I assume it is just for resharper to not scream about some warnings. I think you should consult with @bubus that both of you want to use resharper

Reply:
I think it doesn't matter if @bubus128 wants to use ReSharper. If not, that's okay. He doesn't have to insert these comments then. I'll add them whenever I will be making changes to test classes without these. Moreover, he'll probably use different library for unit tests, one that wouldn't need these comments either way.